content/components/tab-panels.mdx

```diff
@@ -1,14 +1,91 @@
 ---
 title: Tab panels
-description: Use tab panels for tabs with panel navigation.
+description: Tab panels let users switch between views in the same context.
 railsId: Primer::Alpha::TabPanels
 ---
 
 import ComponentLayout from '~/src/layouts/component-layout'
-import {Text} from '@primer/react'
 export default ComponentLayout
+import {Box, Text} from '@primer/react'
 
-<Note variant="warning">
-  <Text sx={{display: 'block', fontWeight: 'bold', mb: 2}}>Work in progress</Text>
-  We are currently working on general guidance for this component.
-</Note>
+<img
+  width="960"
+  role="presentation"
+  src="https://user-images.githubusercontent.com/2313998/231862745-5099ccae-9f65-4a8e-a6f5-cf5c1b6ba3c2.png"
+/>
+
+## Usage
+
+Tab panels are used to group related content into views that are shown and hidden. Each view has a corresponding tab that provides a label and controls the panel's visibility.
+
+Tabs in a tab panel should not be links that navigate the user to a new URL. If you want tab-like navigation, use a [tab nav](/components/tab-nav) instead.
+
+<Box display="flex" flexDirection={['column', 'column', 'column', 'column', 'row']} sx={{alignItems: "center", gap: 3}}>
+  <Box flex={1}>
+
+For example, a tab panel is used to switch between the "Branches" and "Tags" views in the branch selection dialog.
+
+  </Box>
+
+  <img
+    width="456"
+    role="presentation"
+    src="https://user-images.githubusercontent.com/2313998/231862746-9f35f3a6-6f31-4733-b1af-2bad9cd84745.png"
+  />
+</Box>
+
+### Best practices
+
+The same best practices apply to tab panels as they do to [tab nav](/components/tab-nav#best-practices).
+
+## Anatomy
+
+<img
+  width="960"
+  role="presentation"
+  src="https://user-images.githubusercontent.com/2313998/231862747-ea4e4b3b-847f-4ead-8b73-aa9bfc99fb32.png"
+/>
+
+**Tab list:** A set of buttons used to switch between views. Each tab should concisely label the view it's associated with.
+
+**Tab panel:** The view associated with the active tab.
+
+
+## Options
+
+The same options are available to tab panels as they are to [tab nav](/components/tab-nav#options).
+
+## Accessibility
+
+### Keyboard navigation
+
+<Box display="flex" flexDirection={['column', 'column', 'column', 'column', 'row']} sx={{gap: 3}}>
+  <Box flex={1}>
+
+Tabs, like all interactive elements, can be activated using a cursor or keyboard.
+
+When a user moves focus into the `tablist` using the Tab key, the active tab should be focused. A `tablist` is only a single Tab stop.
```

Comment:
```suggestion
When a user moves focus into the `tablist` using the Tab key, the active tab should be focused. A `tablist` is only a single Tab stop. Once focused, navigation through the `tablist` can then conducted with Left, Right, Home, and End keypresses. 
```

Expanded on this a bit to communicate what is expected after the `tablist` receives focus.